antevus-app/fix_all_eslint.py

```
@@ -0,0 +1,142 @@
+
+# Configure ESLint to ignore JS files in scripts directory
+eslintignore = """scripts/
+test-credential-security.js
+"""
+
+with open('.eslintignore', 'w') as f:
+    f.write(eslintignore)
+print("Created .eslintignore")
+
+print("\nRunning ESLint to check remaining issues...")
+result = subprocess.run(['npm', 'run', 'lint'], capture_output=True, text=True)
```

Ruff (0.13.1)

Starting a process with a partial executable path

(S607)